<launch>
    <node name="navigation_docking" pkg="navigation_docking" type="navigation_docking_node" output="screen">
        <rosparam command="load" file="$(find navigation_docking)/config/config.yaml"/>
    </node>
</launch>